src/ndsctl_thread.c

```diff
@@ -295,9 +295,14 @@ ndsctl_handler(int fd)
 static void
 ndsctl_auth(FILE *fp, char *arg)
 {
+	s_config *config = config_get_config();
 	t_client *client;
 	unsigned id;
 	int rc;
+	int seconds = 60 * config->session_timeout;
```

Comment:
Is this meant to mean forever?
I think -1 is a value for that.

Reply:
0 would be forever, but we want this to be the value of client->session_end in seconds.